docs/src/use/getting-started.md

```diff
@@ -20,39 +23,30 @@ To use ESLint, you must have [Node.js](https://nodejs.org/en/) (`^18.18.0`, `^20
 
 You can install and configure ESLint using this command:
 
-```shell
-npm init @eslint/config@latest
-
-# or
-
-yarn create @eslint/config
-
-# or
-
-pnpm create @eslint/config@latest
-```
+{{ npm_tabs({
+    command: "init",
+    packages: ["@eslint/config@latest"],
+    args: []
+}) }}
 
 If you want to use a specific shareable config that is hosted on npm, you can use the `--config` option and specify the package name:
 
-```shell
-# use `eslint-config-standard` shared config
-
-# npm 7+
-npm init @eslint/config@latest -- --config eslint-config-standard
-
-```
+{{ npm_tabs({
+    command: "init",
+    packages: ["@eslint/config@latest", "eslint-config-standard"],
+    args: ["--", "--config"],
+    comment: "use `eslint-config-standard` shared config - npm 7+"
+}) }}
```

Comment:
This compiles to 
![Screenshot 2024-09-24 145055](https://github.com/user-attachments/assets/2b734c26-18f5-49e4-9ec8-e2575f4be842)

can we make it to this
![Screenshot 2024-09-24 144946](https://github.com/user-attachments/assets/0bce251c-c391-462a-b8c3-5c5e6c653492)

Reply:
Here it would not be easy to do because we are rendering all the packages together in the for loop along will the arguments.

Does: `npm init -- --config @eslint/config@latest eslint-config-standard` work same as:
`npm init @eslint/config@latest -- --config  eslint-config-standard` ?

Reply:
for now we can write them in packages key:
```
{{ npm_tabs({
    command: "init",
    packages: ["@eslint/config@latest", "--", "--config" "eslint-config-standard"],
    args: [],
    comment: "use `eslint-config-standard` shared config - npm 7+"
}) }}
```